### PR TITLE
Future work!

### PR DESCRIPTION
**Pull request by VirtualSatai**
This is a first draft, comments please.

### CODE REVIEW

**Comment by VirtualSatai** [comment]
NTP IN FULL

File: cha/epilogue/futurework.tex [hunk 1]
```diff
@@ -1 +1,101 @@
...
+As for the master, the UI does not show if any slaves connect, other than a prompt to give slaves permission, and it does not have an overview over all the slaves connected.
+
+Secondly the stability of the app could be improved in several ways.
+For example we currently only allow slaves to connect prior to starting playback, they should be able to connect at any time as long as the network exists.
+
+Thirdly as previously mentioned, supporting some external sources of music, for example Spotify, would also greatly improve the usability of the app.
+
+\section{Using an Alternative to WiFi Direct}
+In a WiFi Direct network, there is an upper limit on the maximum number of connected devices, theoretically the limit is 254 devices in a group.
+However in practice it is up to the implementation and hardware limitation on the devices in question, realistically the limit is in the six to ten devices range.\tnote{Det er svært at finde en reliable kilde på dette. Wat do?}
```
Comment (TriXXeD) [inline cha/epilogue/futurework.tex]: Use the source you have, it will have to suffice.

**Review by TriXXeD — COMMENTED** [review]
I removed the stereo stuff since stereo isnt really a psychacoustic effect tho. For digital surround sound headsets sound localization(a psychoacoustic effect) is utilized to fake the 5.1/7.1 experience. However for stereo, and surround sounds with the proper tools and setup, surround sound/stereo sound is simply achieved, without psychoacoustic faking it.

Atleast to my knowledge and understanding of the sources we have used stereo and surround sound arent mentioned as effects. If you have sources saying otherwise we can add it back in.

**Comment by TriXXeD** [comment]
I removed the stereo stuff since stereo isnt really a psychacoustic effect tho. For digital surround sound headsets sound localization(a psychoacoustic effect) is utilized to fake the 5.1/7.1 experience. However for stereo, and surround sounds with the proper tools and setup, surround sound/stereo sound is simply achieved, without psychoacoustic faking it.

Atleast to my knowledge and understanding of the sources we have used stereo and surround sound arent mentioned as effects. If you have sources saying otherwise we can add it back in.

**Review by mathiassmichno — APPROVED** [review]

**Pull request merged**